tsdb/exemplar.go

```diff
@@ -202,7 +202,8 @@ Outer:
 }
 
 func (ce *CircularExemplarStorage) ValidateExemplar(l labels.Labels, e exemplar.Exemplar) error {
-	seriesLabels := l.String()
+	var buf [1024]byte
```

Comment:
Do you think it's worth adding a comment explaining why `l.Bytes()` is being used or is that obvious to people more familiar with go than myself?

Reply:
Rather than write a comment (or ~10 of them for all the places that `.Bytes()` is called), I would rename it to something that is more immediately clear.

Based on usage in the PromQL engine, perhaps `Signature()`.